wdfn-server/waterdata/camera.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+
+# proof of concept. should call service to get
+
+JSON_CAMERA_FILE = os.getcwd()+"/data/monitoring_camera_data.json"
+
+
+def get_json_camera_data():
```

Comment:
Consider adding docstrings to this and the other functions in this module.

Reply:
done.